CONTRIBUTING.md

```diff
@@ -71,4 +66,4 @@ Thank you to all our sponsors! (please ask your company to also support this ope
 <a href="https://opencollective.com/parcel/sponsor/6/website" target="_blank"><img src="https://opencollective.com/parcel/sponsor/6/avatar.svg"></a>
 <a href="https://opencollective.com/parcel/sponsor/7/website" target="_blank"><img src="https://opencollective.com/parcel/sponsor/7/avatar.svg"></a>
 <a href="https://opencollective.com/parcel/sponsor/8/website" target="_blank"><img src="https://opencollective.com/parcel/sponsor/8/avatar.svg"></a>
-<a href="https://opencollective.com/parcel/sponsor/9/website" target="_blank"><img src="https://opencollective.com/parcel/sponsor/9/avatar.svg"></a>
\ No newline at end of file
+<a href="https://opencollective.com/parcel/sponsor/9/website" target="_blank"><img src="https://opencollective.com/parcel/sponsor/9/avatar.svg"></a>
```

Comment:
Did I miss something, or are these exactly the same?

Reply:
EOL mismatch between OSes? Sure looks like it.

Reply:
Sh******

Thanks, I'll change that

Reply:
I checked the file, it seems that this file is now encoded using LF. I suppose that it was not the case on master. Do you want me to revert changes on that file?

Reply:
I don’t think it makes any difference